1222-public_profile/research_profile.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Scenarios for Research Profile
+
+    @TEST_NP-4125
+    @test
+    Scenario: Creator opens their Research Profile from My Page
+        Given that the user is logged in as Creator
```

Comment:
Om man er Creator, Curator, eller ikke har noen roller har forsåvidt ingen betydning her. Det er kanskje mest riktig med `Given the user is authenticated`